app/assets/stylesheets/application.sass.scss

```
@@ -81,3 +93,40 @@ $color_app-pale-blue: #ccdff1;
     @extend .govuk-notification-banner__heading;
   }
 }
+
+.app-header__navigation-text {
+  @include nhsuk-font(16);
+  border-bottom: nhsuk-spacing(1) solid transparent;
+  border-top: nhsuk-spacing(1) solid transparent;
+  color: $color_nhsuk-blue;
+  display: block;
+  padding: 12px nhsuk-spacing(3);
+  text-decoration: none;
+
+  @include mq($from: large-desktop) {
+    color: $color_nhsuk-white;
+    line-height: normal;
+  }
+
+  .nhsuk-icon__chevron-right {
+    fill: $color_nhsuk-grey-3;
+    position: absolute;
+    right: nhsuk-spacing(1);
+    top: 11px;
+  }
+}
+
+.nhsuk-header__menu-toggle {
+  @include mq($until: desktop) {
+    right: 0px;
+  }
+}
```

Instead of an override, we could make this new modifier class specific to our app:

```suggestion
.app-header__menu-toggle--no-search {
  @include mq($until: desktop) {
    right: 0px;
  }
}
```

Then we can add it to the menu element in the header. Might have to tweak the rule if it [isn't specific enough](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).

---

Done